workProj.py

```diff
@@ -1,8 +1,59 @@
-import re, csv
+import re, csv, os
+from datetime import datetime, date, time
 
-#1st step. Read the salesData.csv and turn the data into a Dict
```

Comment:
After rearranging code to have functions at the top of file, suggest using if __name__ == "__main__" for the execution portion of this module to make it explicitly clear that you're running things.